setup.py

```diff
@@ -1,17 +1,25 @@
 from setuptools import setup
 
+try:
+    import pypandoc
+    long_description = pypandoc.convert('README.md', 'rst')
+except(IOError, ImportError):
+    long_description = open('README.md').read()
+
```

Comment:
Also, take note of the third solution, which apparently fixes some potential problems with the second solution.